<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <widget class="QTableWidget" name="table_Stats">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>651</width>
     <height>211</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>a_max</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>a_min</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>v_max</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>v_min</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>d_max</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>d_min</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>X</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Z</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rx</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ry</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rz</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration (sec):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Duration">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Units match those in the original text file.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
